torch/_refs/__init__.py

```diff
@@ -6317,6 +6318,7 @@ def select_scatter(x: TensorLikeType, src: TensorLikeType, dim: int, index: int)
 # TODO: This must return a sparse tensor if the input is sparse, but refs have
 # no sparse support. See narrow_copy_sparse in core.
 narrow_copy = _make_copy_from_view(narrow)
+view_copy = _make_copy_from_view(view)
```

Comment:
```suggestion
view_copy = _make_copy_from_view(aten.view)
```

Reply:
Wow, this makes me glad that I resubmitted this in pieces, so that we caught this.

I'll also change the pre-existing `_copy` decompositions...